apps/gpc-verifier/app/hooks/useProofRequest.ts

```diff
@@ -0,0 +1,147 @@
+import { useMemo } from "react";
+import {
+  serializeGPCProofConfig,
+  gpcBindConfig,
+  GPCProofConfig,
+  PODMembershipLists
+} from "@pcd/gpc";
+import { POD_INT_MAX } from "@pcd/pod";
+import { generateSnarkMessageHash } from "@pcd/util";
+
+// https://docs.pcd.team/types/_pcd_gpc.GPCProofConfig.html
+const proofConfig: GPCProofConfig = {
+  // Each POD object mentioned here must be provided in GPCProofInputs.
+  // The names assigned here are used to refer these PODs in GPCProofInputs,
+  // as well as elsewhere in configuration.
+  pods: {
+    govID: {
+      entries: {
+        // Prove the presence of an entry called "idNumber", hide its value,
+        // and also prove that it is not a member of the sanctionsList.
+        idNumber: { isRevealed: false, isNotMemberOf: "sanctionsList" },
+        // Prove the presence of an entry called "firstName", hide its value.
+        firstName: { isRevealed: false },
+        // Prove the presence of an entry called "lastName", hide its value.
+        lastName: { isRevealed: false },
+        // Prove the presence of an entry called "age", hide its value.
+        // and prove that it is >= 18
+        age: {
+          isRevealed: false,
+          inRange: { min: 18n, max: POD_INT_MAX }
+        },
+        // Prove the presence of an entry called "owner", hide its value, and prove
+        // that I own the corresponding Semaphore identity secrets.
+        owner: { isRevealed: false, isOwnerID: true }
+      }
+    },
+    paystub: {
+      entries: {
+        // Prove the presence of an entry called "firstName", hide its value, and
+        // prove that it equals to the firstName in the govID POD.
+        firstName: { isRevealed: false, equalsEntry: "govID.firstName" },
+        // Prove the presence of an entry called "lastName", hide its value, and
+        // prove that it equals to the firstName in the govID POD.
+        lastName: { isRevealed: false, equalsEntry: "govID.lastName" },
+        // There's an entry "currentEmployer" in the paystub POD, because it is
+        // not specified here, it will be ignored, meaning the proof says nothiing
+        // about the entry, and the entry won't be in the revealed claims.
+        // currentEmployer: {}
+        // Prove the presence of an entry called "startDate", reveal its value.
```

Comment:
Should this scenario prove something to indicate Gerry is still employed?  E.g. an endDate or issueDate which is within 1 week of the current date?

Reply:
Good point. I'll add issueDate here.
Having an endDate here is good, but won't entirely solve our problem.
Because when this POD is issued, Gerry could still be employed, and there is no endDate (which can be set to POD_INT_MAX).
But later Gerry can continue reusing this POD even when he is no longer employed.

Reply:
Yeah, for a real-world scenario, you'd probably require the proof of income to be issued recently, like within the past 30 days, which can be checked with another inRange check.

Reply:
FYI, actually I cannot add more GPCProofEntryConfig here. Because I'm getting this error
```Error: There are no circuits with parameters satisfying these requirements: {"nObjects":2,"nEntries":11,"merkleMaxDepth":4,"nNumericValues":3,"nLists":1,"maxListSize":1,"tupleArities":{}}```
I guess we need a bigger circuit?

Reply:
I'm going to remove the firstName and lastName checks, and add SSN fields and check instead, like we discussed.